### PR TITLE
Added getting started with PowerShell module

### PR DESCRIPTION
**Pull request by dcaro**
Added steps for installation from the gallery and local import.

### CODE REVIEW

File: powershell-module/README.md [hunk 1]
```diff
@@ -17,6 +18,24 @@ The Az.Tools.Migration PowerShell module contains cmdlets that perform the follo
...
+
+You can use either method to use this module:
+
+1. Install this module from the [PowerShell gallery](https://www.powershellgallery.com/packages/Az.Tools.Migration).
+
+```
+Install-Module -Name Az.Tools.Migration
+```
+
+1. Clone this repository and import the module locally.
```
Comment (keithbabinec) [inline powershell-module/README.md:31]: This didn't auto-bullet to 2 in the markdown preview, so you might want to specify “2.” on the beginning of the line instead.

**Review by keithbabinec — APPROVED** [review]
Approved with 1 minor suggestion

File: powershell-module/README.md (hunk 1 above)
Comment (dcaro) [reply powershell-module/README.md]: Thanks, I missed this.

**Pull request merged**